src/index.ts

```diff
@@ -44,7 +49,9 @@ export = class OutputDcatUs11 {
         throw new DcatUsError('DCAT-US 1.1 feed template is not provided.', 400);
```

Comment:
We should probably change this error message to account for the feed version number

Reply:
Good point I have changed error message to account for feed version number